src/helpers/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { Todo } from '../types/Todo';
+import { FilterType } from '../types/FilterType';
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  filterType: FilterType,
+  query: string,
+) => {
+  const copyVisibleTodos = [...todos];
+
+  const searchedTodos = copyVisibleTodos.filter(({ title }) => {
+    return title.toLowerCase().includes(query.toLowerCase());
+  });
```

Comment:
not sure about creating copyVisibleTodos  array, since Filter method is immutable